shiny/express/layout.py

```diff
@@ -225,6 +232,111 @@ def layout_column_wrap(
     )
 
 
+def layout_columns(
+    *,
+    col_widths: BreakpointsUser[int] = None,
+    row_heights: BreakpointsUser[CssUnit] = None,
+    fill: bool = True,
+    fillable: bool = True,
+    gap: Optional[CssUnit] = None,
+    class_: Optional[str] = None,
+    height: Optional[CssUnit] = None,
+    **kwargs: TagAttrValue,
+):
+    """
+    Create responsive, column-based grid layouts, based on a 12-column grid.
+
+    Parameters
+    ----------
+    col_widths
```

Comment:
I think we should be documenting `args` here?

Reply:
I'm not sure, it isn't documented in `shiny.express.layout_column_wrap()`, so I also didn't document it for `layout_columns()`.

https://github.com/posit-dev/py-shiny/blob/3bc470f9a3e4ca58749a5968cbbe3f5327d7825f/shiny/express/layout.py#L168-L170